mlshvch/hometask-4/my_hash.rb

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require 'digest'
+
+# class MyHash with custom implementation of Hash Table
+class MyHash
+  include Enumerable
```

Comment:
By the way, I see you included `Enumerable`, but where is the `each` method?

Reply:
I also want to know. Other methods such as each_with_object are available according to MyHash#methods

Reply:
Please read the Enumerable module documentation carefully:
"The class must provide a method each, which yields successive members of the collection."

The presence of other methods does not mean that they actually work, try calling `map` on your hash and you will see smth like `NoMethodError (undefined method `each' for #<MyHash:...>)`

Reply:
Understood